api/controllers/auth.js

```diff
@@ -11,8 +11,13 @@ const login = async (req, res) => {
 };
 
 const googleLogin = async (req, res) => {
-  const response = await services.googleLogin(req.body);
-  res.send(response);
+  const response = await services.googleLogin(req.user);
+  res.redirect('http://localhost:3000/?tokenGoogle=' + response);
```

Comment:
hardcode

Reply:
Use .env variables in this case `const HOST = process.env.HOST || 'http://locahost:3000/';`